src/components/IndvReview.jsx

```diff
@@ -35,6 +40,12 @@ export const IndvReview = () => {
     });
   };
 
+  const handleDelete = (comment_id) => {
+    deleteComment(comment_id).then((comment) => {
+      setComments(comments);
```

Comment:
This is just setting the comments to what they were before and feeding into your problem with the other useEffect